addon/components/o-s-s/copy.hbs

```diff
@@ -1,2 +1,3 @@
-<OSS::Button @icon="far fa-copy" @square={{true}} @size="sm" {{on "click" this.copy}}
+<OSS::Button class={{if this.inline 'oss-copy--inline'}}
+             @icon="far fa-copy" @square={{true}} @size="sm" {{on "click" this.copy}}
              {{enable-tooltip placement="top" title=(t "oss-components.copy.tooltip") trigger="hover"}} />
```

Comment:
IMO it would be better to have a whole if/else here and not override buttons style. And just display an raw icon in a span/div